cdist/conf/type/__cron/gencode-remote

```diff
@@ -26,7 +26,7 @@ command="$(cat "$__object/parameter/command")"
 
 if [ -f "$__object/parameter/raw" ]; then
    raw="$(cat "$__object/parameter/raw")"
-   entry="$raw $command"
+   entry="$raw $command # $name"
```

Comment:
@telmich this '# $name' is here because it is appended for every case except for raw_command. So it should be correct.